core/src/main/scala/org/apache/spark/SparkHadoopWriter.scala

```diff
@@ -105,10 +107,24 @@ class SparkHadoopWriter(@transient jobConf: JobConf)
   def commit() {
     val taCtxt = getTaskContext()
     val cmtr = getOutputCommitter()
+    val conf = SparkEnv.get.conf
+    val dagSchedulerActor =
+      AkkaUtils.makeDriverRef("DAGSchedulerSupervisor/DAGScheduler", conf, SparkEnv.get.actorSystem)
```

Comment:
All of the tests that broke, did so because of "Name is not unique" errors. I don't know Akka nearly as well as I should, but why was this name added explicitly?